### PR TITLE
fix some missing hard-coded paths #647

### PR DESCRIPTION
**Pull request by stineje**
delete some more hard coded items - left RISCVCTG as is in terms of name as its pretty generic.  Put into site-setup.sh

### CODE REVIEW

File: site-setup.sh [hunk 1]
```diff
@@ -30,6 +30,10 @@ export SYN_MW=/home/jstine/MW
 export SYN_memory=/home/jstine/WallyMem/rv64gc/
 #export osumemory=/import/yukari1/pdk/TSMC/WallyMem/rv64gc/
 
+# Environmental variables for FPGA
```
Comment (jordancarlin) [inline site-setup.sh]: Pretty sure CTG is test generation related, not FPGA related

Comment (stineje) [reply site-setup.sh]: fixed..yes..sorry had FPGA on the brain.

**Comment by davidharrishmc** [comment]
Is this ready for merge?

**Comment by stineje** [comment]
All ready ***@***.*** found some more paths and removed.

**Comment by ross144** [comment]
wave_config.wcfg is good.

**Pull request merged**